### PR TITLE
feat:setup the automapper for dtos's in the application layer

### PR DESCRIPTION
**Pull request by erafazliu**

### CODE REVIEW

**Review by ArianitSGashi — APPROVED** [review]
Nicely done!

File: src/OnlineVotingS.Application/Mapping/MappingProfile.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using OnlineVotingS.Application.DTO;
+using OnlineVotingS.Domain.Entities;
+
+
+namespace OnlineVotingS.Application.Mapping;
+    public class MappingProfile : Profile
+    {
+        public MappingProfile()
+        {
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Mapping/MappingProfile.cs]: remove whilte line

File: src/OnlineVotingS.Application/Mapping/MappingProfile.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            //CreateMap<Source, Destination>();
+            CreateMap<Candidates, CandidateDTO>().ReverseMap();
+            CreateMap<Elections, ElectionDTO>().ReverseMap();
+            CreateMap<Votes, VoteDTO>().ReverseMap();
+            CreateMap<Result, ResultDTO>().ReverseMap();
+            CreateMap<Complaints, ComplaintDTO>().ReverseMap();
+            CreateMap<Campaign, CampaignDTO>().ReverseMap();
+            CreateMap<Feedback, FeedbackDTO>().ReverseMap();
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Mapping/MappingProfile.cs]: remove white line

File: src/OnlineVotingS.Application/Mapping/MappingProfile.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        }
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Mapping/MappingProfile.cs]: remove white line

File: src/OnlineVotingS.Application/Mapping/MappingProfile.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
+﻿using AutoMapper;
+using OnlineVotingS.Application.DTO;
+using OnlineVotingS.Domain.Entities;
+
+
+namespace OnlineVotingS.Application.Mapping;
+    public class MappingProfile : Profile
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Mapping/MappingProfile.cs:7]: add a whitle line here please

File: src/OnlineVotingS.Application/Mapping/MappingProfile.cs [hunk 5]
```diff
@@ -0,0 +1,23 @@
+﻿using AutoMapper;
+using OnlineVotingS.Application.DTO;
+using OnlineVotingS.Domain.Entities;
+
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Mapping/MappingProfile.cs:4]: remove white line

**Review by AdhurimUP — APPROVED** [review]

**Pull request merged**